libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -147,6 +151,8 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
      */
     control: FormFieldControl<any>;
 
+    isGroupField: boolean;
```

Comment:
I think this should be with `_` underscore

Reply:
Actually unnecessary property. Removed